Requirements.md

```diff
@@ -0,0 +1,13 @@
+## Software Requirements Specifications for Spring 2020 "Intro to Software Engineering" "Craft Topography" Project at Wright State Univesity
+### Functional Requirements
+  #### User Story A
+   - Req 1.0 Craft implementation shall create a terrain from a topographic image
```

Comment:
I have experience doing this. Let me know if you want help.

Reply:
I will need some help implementing the height data that i have generated. I know where in the code i need to implement my data to change the height of the blocks because i have hardcoded a single value to give one coordinate in game a specific block height. However, i cannot seem to figure out how to implement my data.